stats/members/mdetail.php

```diff
@@ -9,12 +9,20 @@
 include_once('../includes/team.inc');
 include_once('../includes/member.inc');
 
-$tally_name = array_get( $_GET, 'tally_name', null );
-
 $id = get_integer_param($_GET, 'id', null, 0, null);
-
 $user = User::load_from_uid($id);
 
+$valid_tally_names = array_keys($page_tally_names);
```

Comment:
`$page_tally_names` kind of comes out of the blue, so it might be good to add `// $page_tally_names` to line 6.